### ISSUE TITLE
[Bug]:Subtitles do not consistently appear throughout video

### ISSUE THREAD

**Issue by dulledtalons** [FreeTubeApp/FreeTube#8980]
### Guidelines

- [x] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [x] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [x] I have [searched the issue tracker for open and closed issues](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+sort%3Arelevance-desc) that are similar to the bug report I want to file, without success.
- [x] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [x] This issue contains only one bug.
- [x] I have read and agree to follow the [rules](https://docs.freetubeapp.io/community/rules/).

### Describe the bug

For videos with official uploaded captions: instead of captions switching to the next line, the next lines will not display at all, but the following lines of captions WILL display. In short, each “page” of captions displays alternatingly. 

### Expected Behavior

Captions switch smoothly to the next line without disappearing.

### Issue Labels

accessibility issue, content not loading

### FreeTube Version

v0.24.0

### Operating System Version

Windows 11 version 25H2

### Installation Method

.exe

### Primary API used

Local API

### Last Known Working FreeTube Version (If Any)

v0.23.15

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by absidue** [comment]
Could you please provide a link to a video that is affected by this issue?

**Comment by dulledtalons** [comment]
> Could you please provide a link to a video that is affected by this issue?

Currently it seems to affect all videos I watch with creator-uploaded captions?

[Example of what I mean](https://file.garden/aPORi5oGSn0LJJpj/ExampleVideo.mp4) using [this video](https://youtu.be/hEfyX4Wl8aM).

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Please retry to reproduce this issue. Im unable to reproduce this as im seeing all the subs appearing correctly

**Comment by dulledtalons** [comment]
> Please retry to reproduce this issue. Im unable to reproduce this as im seeing all the subs appearing correctly

I tested some more, and it seems this actually only happens when the video is fullscreen?

[Recording of this happening](https://file.garden/aPORi5oGSn0LJJpj/SubtitlesNotWorking.mp4), with [this video](https://youtu.be/V563dGxDiaI). Sorry for the longer clip but I hope this clarifies the issue.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @dulledtalons im still unable to reproduce this behavior. Do you have another device you can test this on to see if it also happens on there?

**Comment by dulledtalons** [comment]
> Hi [@dulledtalons](https://github.com/dulledtalons) im still unable to reproduce this behavior. Do you have another device you can test this on to see if it also happens on there?

Hello, sorry for the late response. I was able to test on a different Windows PC, using the same install method of using the x64 setup.exe, and the subtitles appeared normally there. On my main PC, I uninstalled Freetube and deleted its AppData folder before reinstalling and trying a video on the clean reinstall. However, I still experienced the issue of subtitles appearing intermittently.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Good to know. The fact that it works on another machine but doesnt work on this one seems to indicate that it is specific to your device and not an issue that everybody is having. 

Could you try to update your graphics drivers and make sure your OS is up to date. Sometimes doing this resolves weird issues like this.

I will close this issue though because its an isolated issue but feel free to keep us posted on this
